packages/perseus-editor/src/widgets/phet-simulation-editor.tsx

```diff
@@ -31,6 +32,13 @@ class PhetSimulationEditor extends React.Component<Props> {
         };
     }
 
+    getSaveWarnings(): ReadonlyArray<string> {
+        if (makeSafeUrl(this.props.url, "en") === null) {
+            return ["Please enter a URL from the PhET domain."];
+        }
+        return [];
+    }
+
     render(): React.ReactNode {
```

Comment:
Make sure to test this locally in webapp/ a znd before deploying!

Reply:
We have just discovered that local dev server doesn't allow for content editing. I will make a ZND instead!

Reply:
Done! Looks good.